react/.storybook/webpack.config.js

```diff
@@ -1,6 +1,16 @@
 const path = require("path");
 
 module.exports = (baseConfig, env, defaultConfig) => {
+  defaultConfig.module.rules.forEach((rule, ruleIndex) => {
+    if (rule && typeof rule !== 'string' && rule.test.toString().indexOf('js') > -1) {
+      if (defaultConfig.module.rules[ruleIndex].options) {
```

Comment:
can this be simplified?
```suggestion
      if (rule.options) {
```

Reply:
same to the ones below